scripts/browserstack_tests/browserstack_frontend_test1.rb

```diff
@@ -0,0 +1,39 @@
+require 'rubygems'
```

Comment:
@sentientmachine This is great!  Let's add a sentence or two describing what this is, what it is for, and how people can use it.

There's also some commented out lines, could we remove them?